lib/mix/lib/mix/tasks/profile.eprof.ex

```diff
@@ -194,7 +200,7 @@ defmodule Mix.Tasks.Profile.Eprof do
     end
 
     :eprof.start()
-    :eprof.profile([], fun, Keyword.get(opts, :matching, {:_, :_, :_}))
+    {:ok, return_value} = :eprof.profile([], fun, Keyword.get(opts, :matching, {:_, :_, :_}))
```

Comment:
This bit is potentially dangerous, as it might return `{:error, Reason}` https://www.erlang.org/doc/man/eprof.html#profile-1

> If tracing could be enabled for P and all processes in Rootset, the function returns {ok,Value} when Fun()/apply returns with the value Value, or {error,Reason} if Fun()/apply fails with exit reason Reason. Otherwise it returns {error, Reason} immediately.

It would be a rather hard failure here, on the other hand I'm not sure a possible error was handled gracefully before. Also not sure how to best "gracefully" handle it here.

Feedback welcome!